server/build.gradle

```diff
@@ -62,6 +62,7 @@ dependencies {
     annotationProcessor "org.projectlombok:lombok:${lombokVersion}"
     testCompileOnly "org.projectlombok:lombok:${lombokVersion}"
     testAnnotationProcessor "org.projectlombok:lombok:${lombokVersion}"
+    testImplementation project(":lh-test")
```

Comment:
we are going to name it lh-test? is so go ahead, otherwise i would like to merge it with a definitive name

Reply:
Well...it is for "test" and it is for "lh"..